_includes/scripts.html

```diff
@@ -9,6 +9,6 @@
 <script src="{{ _src | relative_url }}"></script>
 {% endfor %}{% endfor %}
 {% if layout.includes_touchpoints %}
-  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js" integrity="sha256-bJJrp2Fye5tHOooPte/au2FqGDZwmymjr4+r4UrbeaE="  crossorigin="anonymous"></script>
+  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js" async></script>
```

Comment:
I think we want this to be synchronous since the `touchpoints_translations` script needs to make sure it runs after touchpoints loads?
```suggestion
  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js"></script>
```

Reply:
Ah you are right, that was the initial implementation.